ee/localserver/server.go

```diff
@@ -82,7 +80,15 @@ func New(ctx context.Context, k types.Knapsack, presenceDetector presenceDetecto
 		return nil, err
 	}
 
-	ecKryptoMiddleware := newKryptoEcMiddleware(k.Slogger(), ls.myLocalDbSigner, *ls.serverEcKey, presenceDetector)
+	munemo, err := getMunemoFromEnrollSecret(k.EnrollSecret())
```

Comment:
I think we might need `k.ReadEnrollSecret()` instead of `k.EnrollSecret()` here, since usually we set the enroll secret via file instead of command-line opt (looks like the previous iteration of this code used `k.ReadEnrollSecret()` too)

Reply:
good catch, updated!